src/coreclr/pal/inc/pal.h

```diff
@@ -2100,10 +2100,10 @@ typedef struct _KNONVOLATILE_CONTEXT_POINTERS {
 
 #elif defined(HOST_RISCV64)
 
-#error "TODO-RISCV64: review this when src/coreclr/pal/src/arch/riscv64/asmconstants.h is ported"
+// #error "TODO-RISCV64: review this when src/coreclr/pal/src/arch/riscv64/asmconstants.h is ported"
```

Comment:
Feel free to delete these now that they are reviewed. We added them so next person don't forget to review them (otherwise they are hard to locate).

Reply:
I removed. Thank you!